DaySelector/tests/DaySelector-specs.js

```diff
@@ -1,157 +1,126 @@
-import {mount} from 'enzyme';
+import '@testing-library/jest-dom';
+import {fireEvent, render, screen} from '@testing-library/react';
 
 import DaySelector from '../DaySelector';
 
 const tap = (node) => {
-	node.simulate('mousedown');
-	node.simulate('mouseup');
-	node.simulate('click');
+	fireEvent.mouseDown(node);
+	fireEvent.mouseUp(node);
+	fireEvent.click(node);
 };
 
 describe('DaySelector', () => {
+	test('should set selected prop to true for the item that is selected by default', () => {
+		render(<DaySelector defaultSelected={0} />);
+		const daySelector = screen.getAllByRole('checkbox')[0].children.item(0).children.item(0);
```

Comment:
The parent also has the selected class and is more relevant for the test
```suggestion
		const daySelector = screen.getAllByRole('checkbox')[0];
```